File: Portal/Entities/ECompany.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using CarrinhoVirtual.Codes;


namespace CarrinhoVirtual.Entities {
    [RestInPeaceEntity]
    [Table("companies")]
    public class ECompany {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Phone { get; set; }
        public string Mobile { get; set; }
        public DateTime CreationDateUTC { get; set; }
        public DateTime ModificationDateUTC { get; set; }
    }
}

File: CarrinhoFlutter/lib/entities/restinpeace/ecompany.dart
//region imports
//author Bruno Tezine
import 'package:json_annotation/json_annotation.dart';
import 'package:decimal/decimal.dart';
part 'ecompany.g.dart';
//endregion

@JsonSerializable(nullable: true)
class ECompany{
	String id;
	String name;
	String email;
	String password;
	String phone;
	String mobile;
	DateTime creationDateUTC;
	DateTime modificationDateUTC;

	ECompany({
		this.id,
		this.name,
		this.email,
		this.password,
		this.phone,
		this.mobile,
		this.creationDateUTC,
		this.modificationDateUTC
});

	factory ECompany.fromJson(Map<String, dynamic> json) => _$ECompanyFromJson(json);
	Map<String, dynamic> toJson() => _$ECompanyToJson(this);
}